Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM docker-registry.nine.ch/ninech/ruby:trusty
```

Comment:
Could we switch this to an official image?

Reply:
Yes. I'll make two PRs: one for Docker, the other one for the integration specs.